app/views/candidate_interface/subject_knowledge/show.html.erb

```diff
@@ -26,13 +26,27 @@
     <% end %>
   <% end %>
 
-  <%= render CandidateInterface::SubjectKnowledgeReviewComponent.new(application_form: @application_form) %>
+  <% if !@subject_knowledge_form.valid? %>
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <div class="govuk-inset-text app-inset-text--error govuk-!-margin-top-0">
+          <h2 class="app-inset-text__title">There is a problem</h2>
+          <p class="govuk-body"><%= @subject_knowledge_form.errors.messages.first[1][0] %></p>
```

Comment:
What happens if there is more than one validation error? Is it possible to use `f.govuk_error_summary`?

Reply:
I couldn’t use `govuk_error_summary` as that creates links to the fields, expecting them to be on the same page. But in this instance it’s on a different page, and we don’t need links as that’s already on the page below.

In theory there should only ever be 1 error as there’s only 1 field involved - but maybe the code should still display them all?

Reply:
Makes sense. I guess we might have another kind of validation in the future but I don't think we should hold this up.